legacy/src/main/java/org/opensearch/sql/legacy/utils/QueryDataAnonymizer.java

```diff
@@ -38,8 +39,9 @@ public static String anonymizeData(String query) {
                     .replaceAll("false", "boolean_literal")
                     .replaceAll("[\\n][\\t]+", " ");
         } catch (Exception e) {
-            LOG.warn("Caught an exception when anonymizing sensitive data");
-            resultQuery = query;
+            LOG.warn("Caught an exception when anonymizing sensitive data.");
+            LOG.debug("String {} failed anonymization.", query);
```

Comment:
You still print the query on failure in the log.

Reply:
Only in debug level which shouldn't be enabled in a production build right?